src/hooks/routing/useAutoRouterSupported.ts

```diff
@@ -0,0 +1,8 @@
+import useActiveWeb3React from 'hooks/useActiveWeb3React'
+
+import { AUTO_ROUTER_SUPPORTED_CHAINS } from './clientSideSmartOrderRouter'
+
+export default function useAutoRouterSupported(): boolean {
```

Comment:
This can be exported as a hook directly from `./clientSideSmartOrderRouter`, no need for a new file.